internal/otelcollector/config/metric/gateway/config_builder.go

```diff
@@ -175,6 +189,11 @@ func makeServicePipelineConfig(pipeline *telemetryv1alpha1.MetricPipeline, opts
 
 	input := pipeline.Spec.Input
 
+	// Perform the transform before runtime resource filter as InstrumentationScopeK8sCluster is required for dropping container/pod metrics
```

Comment:
update the comment to be `InstrumentationScopeRuntime ` instead of `InstrumentationScopeK8sCluster`